app/src/main/java/com/celzero/bravedns/ui/SettingsFragment.kt

```diff
@@ -485,10 +494,9 @@ class SettingsFragment : Fragment(R.layout.activity_settings_screen) {
     }
 
     private fun refreshOnDeviceBlocklistStatus() {
-        if (!persistentState.blocklistFilesDownloaded || !persistentState.blocklistEnabled || persistentState.numberOfLocalBlocklists <= 0) {
+        if (!persistentState.blocklistEnabled) {
```

Comment:
```
if (isPlayStoreFlavour()) {
```

Reply:
Modified the call from onResume to `refreshOnDeviceBlocklistUi` which handles `playStoreFlavour` check